chat-plugins/poll.js

```diff
@@ -362,6 +363,7 @@ exports.commands = {
 			if (room.poll.pollArray[num].pollNum === parseInt(target)) delete room.poll.pollArray[num];
 
 			return this.privateModCommand("(A poll was ended by " + user.name + ".)");
+			activePolls = activePolls - 1;
```

Comment:
scrap the activePolls part and change the ``delete room.poll.pollArray[num]`` to this ``room.poll.pollArray.splice(num, 1)`` on like 363

Reply:
Also do this for timer on line 326 (it wont let me review there kek)